tools/src/main/scala/scala/scalanative/linker/Linker.scala

```diff
@@ -42,6 +42,8 @@ final class Linker(dotpath: Option[String], paths: Seq[String]) {
     }
 
   def link(entry: Global): (Seq[Global], Seq[Defn]) = {
+    println(s"Linking $entry")
```

Comment:
This and all the other `println`-s in Linker need to be removed. We need to think of proper logging implementation instead (https://github.com/scala-native/scala-native/issues/119) but that's out of scope for this pr.